model/src/main/kotlin/config/AnalyzerConfiguration.kt

```diff
@@ -43,5 +43,12 @@ data class AnalyzerConfiguration(
     /**
      * Configuration of the SW360 package curation provider.
      */
-    val sw360Configuration: Sw360StorageConfiguration? = null
+    val sw360Configuration: Sw360StorageConfiguration? = null,
+
+    /**
+     * Enable the usage of project-local package-curations from the [RepositoryConfiguration]. If set to true apply
+     * package-curations from a local .ort.yml file. These package-curations are merged with package-curations from
+     * the command line option, while the one from .ort.yml take precedence.
+     */
+    val applyRepositoryPackageCurations: Boolean = false
```

Comment:
How about `enablePackageCurationsInRepositoryConfiguration`?

I believe that this option potentially applies to all components other than the analyzer.
IIRC the evaluator also has an option to re-apply curations.
So how about moving that to `OrtConfiguration`?

I presume in another PR you'll add a similar flag for `PackageConfigurations`.
These are not used by the analyzer, but evalutor / reporters. I guess these would fit into `OrtConfigurations` so
the option introduced here would fit right next to it?!

Reply:
> How about `enablePackageCurationsInRepositoryConfiguration`?

I like the shorter `applyRepositoryPackageCurations` more.
However as the `.ort.yml` is called "repository configuration" in all the docs, I'll change it.

Reply:
The reason for my proposal was that I was afraid some pick-up "RepositoryPackageCuration" as new term.
Not sure if it's a valid concern.

Reply:
Another way to avoid that could be naming the variable `ignoreLocalPackageCurations`. What do you think?

Reply:
> Another way to avoid that could be naming the variable `ignoreLocalPackageCurations`. What do you think?

I don't like to turn around the logic when using `ignore`, because it will introduce double negative when turning this feature on.

Reply:
> The reason for my proposal was that I was afraid some pick-up "RepositoryPackageCuration" as new term.
> Not sure if it's a valid concern.

`LocalPackageCurations` would also introduce a new term.
I think as `Repository Configuration` is already a term used in multiple places, `repository curation` is IMO self-explanatory.

What do you think about @sschuberth's suggestion `enableRepositoryPackageCurations` [here](https://github.com/oss-review-toolkit/ort/pull/4430#discussion_r702615271)?

Reply:
Ok with me. I don't have a suggestion which solves the issue I outlined :-(

Reply:
> What do you think about @sschuberth's suggestion `enableRepositoryPackageCurations` [here](https://github.com/oss-review-toolkit/ort/pull/4430#discussion_r702615271)?

Ok with me. I don't have a suggestion which solves the issue I outlined :-(

Reply:
PS: I was also thinking about "local" vs. "repository", and while I like "local" for being the more generic term, and ORT *used* to work fine for analyzing local non-VCS directories, that [does not really true hold anymore](https://github.com/oss-review-toolkit/ort/issues/2579#issuecomment-913166262), and we implicitly assume local directories to always be VCS working trees. Also, the term "repository" is already used e.g. in `RepositoryConfiguration`, which otherwise should also be renamed to `LocalConfiguration` or `ProjectConfiguation` (the latter actually is what I wanted to discuss in the ORT Developer Meeting for quite some time, see the [wiki](https://github.com/oss-review-toolkit/ort/wiki/Developer-Meeting)).